tests/e2e-targetallocator/targetallocator-prometheuscr/02-install.yaml

```diff
@@ -0,0 +1,154 @@
+---
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: prometheus-cr-v1beta1
+spec:
+  config:
+    receivers:
+      prometheus:
+        config:
+          scrape_configs: []
+
+    processors:
+
+    exporters:
+      prometheus:
+        endpoint: 0.0.0.0:9090
+    service:
+      pipelines:
+        metrics:
+          receivers: [prometheus]
+          exporters: [prometheus]
+  mode: statefulset
+  serviceAccount: collector
+  targetAllocator:
+    enabled: true
+    prometheusCR:
+      enabled: true
+      scrapeInterval: 1s
+      scrapeConfigSelector: {}
+      probeSelector: {}
+    serviceAccount: ta
+---
+apiVersion: monitoring.coreos.com/v1alpha1
+kind: ScrapeConfig
+metadata:
+  name: scrape-config-cr
+spec:
+  kubernetesSDConfigs:
+    - role: Node
+---
+apiVersion: monitoring.coreos.com/v1
+kind: Probe
+metadata:
+  name: blackbox-exporter
+spec:
+  jobName: http-get
+  interval: 60s
+  module: http_2xx
+  prober:
+    url: blackbox-exporter.monitoring-system.svc:19115
+    scheme: http
+    path: /probe
+  targets:
+    staticConfig:
+      static:
+        - https://example.com
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: check-metrics-cr
+spec:
+  template:
+    spec:
+      restartPolicy: OnFailure
+      containers:
+        - name: check-metrics
+          image: curlimages/curl
+          args:
+            - /bin/sh
+            - -c
+            - |
+              for i in $(seq 30); do
+                if curl -m 1 -s http://prometheus-cr-v1beta1-collector:9090/metrics | grep "otelcol"; then exit 0; fi
```

Comment:
This doesn't look like it checks a metric scraped from either the Probe or ScrapeConfig you added.

Reply:
yes, I will remove it

Reply:
removed